tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -150,8 +150,13 @@ describe('Lobby - P0 Perspective', () => {
     });
   });
   it.skip('[Missing Feature] Loads lobby after page refresh', () => {
+    // Reload the page to ensure session remains
     cy.reload();
-    expect(true).to.eq(false, 'Empty Test');
+    cy.contains('h1', 'Lobby for Test Game');
+    cy.get('#logo');
+    cy.contains('button.v-btn', 'EXIT');
+    cy.contains('button.v-btn', 'READY');
+    cy.get('[data-cy=nav-drawer]').should('not.be.visible');
   });
```

Comment:
The lobby keeps you logged in between refreshes now and it happens in that initial requestStatus check prior to Vue mounting 🙃